start-docker-compose.sh

```diff
@@ -45,15 +46,19 @@ while getopts ':dtzlh' OPTION; do
     z)
       echo "Pulling latest ZAC Docker Image ..."
       startZac=true
+      pullZac=true
       docker compose pull zac
       ;;
     b)
+      [ "$pullZac" = "true" ] && echo "Both -z and -b are specified. Ignoring latest ZAC image!"
```

Comment:
I would rather say that we don't support this combination of options. We should just exit the script if someone specifies both -z and -b I would say. Same with -z and -l.

Better to make this into a separate little PR maybe?

Reply:
Yeah. Will extract the start script